QuickTests/FunctionalTests/DescribeTests.swift

```diff
@@ -0,0 +1,20 @@
+import XCTest
+@testable import Quick
+import Nimble
+
+class DescribeTests: XCTestCase {
+    
+    func testDescribeErrorsIfExampleGroupsAreNotInstantiated() {
+        
+        // fiddling with global state in this test to test a singleton, what could go wrong?
+        let exampleGroup = World.sharedWorld.currentExampleGroup
+        World.sharedWorld.currentExampleGroup = nil
```

Comment:
Messing with the `World` singleton without knowledge of whatever spec is currently using it feels dangerous.